docs/usage/shoot-operations/secretbinding-to-credentialsbinding-migration.md

```diff
@@ -0,0 +1,70 @@
+---
+title: SecretBinding to CredentialsBinding Migration
+---
+
+# SecretBinding to CredentialsBinding Migration
+
+With the introduction of the [`CredentialsBinding`](../../api-reference/security.md#credentialsbinding) resource a new way of referencing credentials through the `Shoot` was created.
+While `SecretBinding`s can only reference `Secret`s, `CredentialsBinding`s can also reference `WorkloadIdentity`s which provide an alternative authentication method.
+`WorkloadIdentity`s do not directly contain credentials but are rather a representation of the workload that is going to access the user's account.
+
+As `CredentialsBinding`s cover the functionality of `SecretBinding`s, the latter are considered legacy and will be deprecated in the future.
+This incurs the need for migration from `SecretBinding` to `CredentialsBinding` resources.
```

Comment:
Does it make sense to add a statement about migrating from `SecretBinding` to a `WorkloadIdentity` based `CredentialsBinding`? Users might be interested in this "direct" approach while they're already taking the effort.

Reply:
I can link the other documentation here. Direct migration from `Secret` to `WorkloadIdentity` is not allowed. Users would still have to go through the original `Secret`.

Reply:
> Direct migration from Secret to WorkloadIdentity is not allowed. Users would still have to go through the original Secret.

That's probably exactly the information that would be interesting for the reader 👍

Reply:
Yes, should be addressed with https://github.com/gardener/gardener/pull/12296/commits/4318d278fd4083181fa1ab68ea9ac64ecb9fa318